src/components/common/CreateUserWizard/CreateUserWizard.tsx

```diff
@@ -5,23 +5,15 @@ import { Props as FormattedMessageProps } from 'react-intl/src/components/messag
 import Heading from '~shared/Heading';
 import ColonyButton from '~shared/Button';
 import WizardTemplate from '~frame/WizardTemplateColony';
-import { withWizard } from '~shared/Wizard';
-import { EmailPermissions } from '~gql';
+import withWizard from '~shared/Wizard/withWizard';
 
+import { FormValues, initialValues } from './validation';
 import StepUserName from './StepUserName';
 import StepUserEmail from './StepUserEmail';
 
 import styles from './CreateUserWizard.css';
 
-export interface FormValues {
```

Comment:
Tried to import these into `./validation`, but results were `undefined`. Unclear why. Importing from `./validation` works no prob.

Reply:
That's weird, as the loader should be able to handle that. Anyway, your solution is good enought.